src/CMakeLists.txt

```diff
@@ -1,15 +1,29 @@
 ﻿# Axel '0vercl0k' Souchet - January 22 2022
-cmake_minimum_required (VERSION 3.8)
+# Note: the cmake version must be the same as in python/pyproject.toml::tool.scikit-build:cmake.minimum-version
+cmake_minimum_required(VERSION 3.20)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_POSITION_INDEPENDENT_CODE True)
 
-project(udmp-parser)
-add_compile_options(
-    $<$<CXX_COMPILER_ID:MSVC>:/W4$<SEMICOLON>/WX>
-    $<$<CXX_COMPILER_ID:GNU>:-Wall$<SEMICOLON>-Wextra$<SEMICOLON>-pedantic$<SEMICOLON>-Werror>
-    $<$<CXX_COMPILER_ID:Clang>:-Wall$<SEMICOLON>-Wextra$<SEMICOLON>-pedantic$<SEMICOLON>-Werror>
+project(
+    udmp-parser
+    DESCRIPTION "A Windows user minidump C++ parser library."
```

Comment:
Should the description reflect the change in the README - 'A Cross-Platform C++ parser library for Windows user minidumps'?

Reply:
Fixed